core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -140,6 +153,26 @@ public LogicalPlan visitRelation(Relation node, AnalysisContext context) {
     return new LogicalRelation(tableName, table);
   }
 
+  private LogicalPlan handleCatalogInfoTable(Relation node, AnalysisContext context) {
```

Comment:
is it possible to return a LogicalRelation instead of LogicalValues? then we could avoid duplicate code in here.